### PR TITLE
apfelnetzwerk

### PR DESCRIPTION
**Pull request by Apfelnetzwerk**
## Type of change

- [X] Add new directory for a server
- [ ] Update directory of a server
- [ ] Documentation Update

## Checklist

- [X] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [X] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [X] I have created the manifest.json with the required values

### CODE REVIEW

**Comment by PartikelTTV** [comment]
seem's legit

File: minecraft_servers/apfelnetzwerk/manifest.json [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      "name": "Target",
+      "color": "#ff6060"
+    },
+    "Community": {
+      "name": "Community",
+      "color": "#c600a5"
+    }
+  },
+  "discord": {
+    "server_id": "851012884674641973"
```
Comment (JakobBruening) [inline minecraft_servers/apfelnetzwerk/manifest.json:74]: Remove quotes, as it is the case in our example. Currently, it is a string containing your server id.
https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#example-manifestjson-file

File: minecraft_servers/apfelnetzwerk/manifest.json [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  "social": {
+    "web": "https://apfelnetzwerk.de/",
+    "web_shop": "https://shop.apfelnetzwerk.de/",
+    "web_support": "https://support.apfelnetzwerk.de/",
+    "twitter": "Apfelnetzwerk",
+    "instagram": "Apfelnetzwerk",
+    "discord": "https://discord.gg/apfelnetzwerk"
+  },
+  "gamemodes": {
+    "BedWars": {
```
Comment (scrummer) [inline minecraft_servers/apfelnetzwerk/manifest.json:20]: change all the keys to lower-case (the name can stay like this) - for every gamemode

**Review by JakobBruening — CHANGES_REQUESTED** [review]

File: minecraft_servers/apfelnetzwerk/manifest.json [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+  "server_wildcards": [
+    "%.apfelnetzwerk.de"
+  ],
+  "supported_languages": [
+    "de"
+  ],
+  "social": {
+    "web": "https://apfelnetzwerk.de/",
+    "web_shop": "https://shop.apfelnetzwerk.de/",
+    "web_support": "https://support.apfelnetzwerk.de/",
```
Comment (scrummer) [inline minecraft_servers/apfelnetzwerk/manifest.json:14]: Remove the trailing slashes for all social domains

**Review by scrummer — CHANGES_REQUESTED** [review]

**Comment by scrummer** [comment]
Looks good, thank you!

**Pull request merged**